eng/pipelines/build-unix-job.yml

```diff
@@ -0,0 +1,46 @@
+# Build on windows desktop
```

Comment:
Is this on windows? It looks like we're passing in a linux-vm name?

Reply:
Blarg, copy pasta comment error. Will fix.